ycmd/completers/typescript/typescript_completer.py

```diff
@@ -247,6 +250,20 @@ def _GetType( self, request_data ):
       info = self._ReadResponse( seq )[ 'body' ]
       return responses.BuildDisplayMessageResponse( info[ 'displayString' ] )
 
+  def _GetDoc( self, request_data ):
+    with self._lock:
+      self._Reload( request_data )
+      seq = self._SendRequest( 'quickinfo', {
+        'file':   request_data[ 'filepath' ],
+        'line':   request_data[ 'line_num' ],
+        'offset': request_data[ 'column_num' ]
+      })
+
+      info = self._ReadResponse( seq )[ 'body' ]
+      message = '{0}\n\n{1}'.format( info[ 'displayString' ],
+                                     info[ 'documentation' ] )
+      return responses.BuildDetailedInfoResponse( message )
+
```

Comment:
I'm happy to change this. Though it is worth noting that in this particular file, i'm just copying the existing style, so I have updated the whole file. Arguably this is a distracting unrelated change, so I hope Val doesn't mind :)